### PR TITLE
Aliases: Add more aliases

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by Zarel** [comment]
I assume you see the error.

File: data/aliases.js [hunk 1]
```diff
@@ -519,6 +523,7 @@ exports.BattleAliases = {
 	"hera": "Heracross",
 	"hippo": "Hippowdon",
 	"honch": "Honchkrow",
+	"kanga": "Kangakhan",
```
Comment (sparkychildcharlie) [inline data/aliases.js]: Kangaskhan

File: data/aliases.js [hunk 2]
```diff
@@ -489,6 +489,7 @@ exports.BattleAliases = {
 	"amph": "Ampharos",
 	"azu": "Azumarill",
 	"bliss": "Blissey",
+	"birdjesus": "Pidgeot",
```
Comment (HoeenCoder) [inline data/aliases.js]: Dosen't `birdjesus` usually refer to Pidgeot-Mega? (correct me if i'm wrong)

Comment (DeathlyPlays) [inline data/aliases.js]: No, the Twitch streamer used it in the Past Gens and that's where it was named but I can make it mega if wanted

**Comment by DeathlyPlays** [comment]
I think that's all the aliases I can think of @Zarel

**Comment by DeathlyPlays** [comment]
Any suggestions?

File: data/aliases.js [hunk 3]
```diff
@@ -637,6 +639,7 @@ exports.BattleAliases = {
 	"se": "Stone Edge",
 	"spin": "Rapid Spin",
 	"sr": "Stealth Rock",
+	"st": "Spectral Thief",
```
Comment (xfix) [inline data/aliases.js]: Does Spectral Thief deserve an alias, considering it's a signature move of one Pokemon, and ST I would rather think is Sleep Talk.

Comment (DeathlyPlays) [inline data/aliases.js]: ah I hear ST for Spectral Thief often

Comment (DeathlyPlays) [inline data/aliases.js]: Especially with the recent release of Marshadow ofc

Comment (Zarel) [inline data/aliases.js]: I wouldn't; it usually means Sleep Talk.

Comment (DeathlyPlays) [inline data/aliases.js]: Oh ok

**Comment by Zarel** [comment]
I wouldn't make it either, specifically.

Honestly, what should really go on is that we should import the PSdex search system.

File: data/aliases.js (hunk 3 above)
Comment (xfix) [inline data/aliases.js]: Context sensitive, but i suspect that when Marshadow stops to be newest thing, ST will stop meaning Spectral Thief.

**Pull request merged**